Gems/Atom/RPI/Code/Source/RPI.Public/RenderPipeline.cpp

```diff
@@ -364,6 +364,7 @@ namespace AZ
                 if (m_scene)
                 {
                     SceneNotificationBus::Event(m_scene->GetId(), &SceneNotification::OnRenderPipelinePassesChanged, this);
+                    SceneRequestBus::Event(m_scene->GetId(), &SceneRequest::PipelineStateLookupNeedsRebuild);
```

Comment:
Can someone at Amazon (maybe @antonmic or @VickyAtAZ) give their insight on this? Would rebuilding the lookup on every pipeline change be an issue?